### PR TITLE
LPD-45015 Adapt ddm contributor to handle localized value

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
Related issue https://liferay.atlassian.net/browse/LPD-45015

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+public class DDMFormFieldTemplateContextContributorUtilTest
+	extends BaseDDMFormFieldTypeSettingsTestCase {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Test
+	public void testGetLocalesMap() {
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtilTest.java:34]: remember to modify the test name too

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+			"availableLocales");
+		JSONObject defaultLocaleJSONObject = (JSONObject)parameters.get(
+			"defaultLocale");
+		JSONObject editingLocaleJSONObject = (JSONObject)parameters.get(
+			"editingLocale");
+
+		Assert.assertEquals(
+			availableLocales.size(), availableLocalesJSONArray.length());
+		Assert.assertEquals("en_US", defaultLocaleJSONObject.get("localeId"));
+		Assert.assertEquals("en_US", editingLocaleJSONObject.get("localeId"));
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtilTest.java:57]: Could we use `LocaleUtil.US` here? :thinking:

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by pedro-oliveira446** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtilTest.java (hunk 2 above)
Comment (pedro-oliveira446) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtilTest.java]: Done!

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtilTest.java (hunk 1 above)
Comment (pedro-oliveira446) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtilTest.java]: Done!

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/localizable/text/LocalizableTextDDMFormFieldTemplateContextContributorTest.java [hunk 3]
```diff
@@ -52,8 +51,8 @@ public class LocalizableTextDDMFormFieldTemplateContextContributorTest
 	public void setUp() throws Exception {
 		super.setUp();
 
+		setUpJSONFactoryUtil();
```
[the remark this answers is not in the document]
Comment (pedro-oliveira446) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/localizable/text/LocalizableTextDDMFormFieldTemplateContextContributorTest.java]: Done!

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtil.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * @author Pedro Leite
+ */
+public class DDMFormFieldTemplateContextContributorUtil {
+
+	public static Map<String, Object> getLocalesMap(Locale defaultLocale) {
+		JSONObject localeJSONObject = _getLocaleJSONObject(defaultLocale);
```
[the remark this answers is not in the document]
Comment (pedro-oliveira446) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtil.java]: Done!

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:forms

**Comment by pedro-oliveira446** [comment]
ci:report:100517450

**Comment by pedro-oliveira446** [comment]
ci:report:100525736

**Comment by pedro-oliveira446** [comment]
ci:report:100525736

**Comment by pedro-oliveira446** [comment]
ci:forward:force

**Pull request closed without merging**